packages/ingest/test/test-SftpProviderClient.js

```diff
@@ -67,7 +73,14 @@ test('SftpProviderClient supports plaintext usernames and passwords', async (t)
     encrypted: false
   });
 
-  const files = await sftpProviderClient.list('/');
+  let files;
```

Comment:
Can we not use the provider client set up in the context?

Reply:
No, because we're testing different types of authentication in this test and a few others, so we need different provider clients rather than using the default one.